### PR TITLE
Bugs 3

### LINKED ISSUE #160: Remove commercial price

### PR DESCRIPTION
**Pull request by rasensio1**
closes #160

### CODE REVIEW

File: app/controllers/photos_controller.rb [hunk 1]
```diff
@@ -7,6 +7,6 @@ def show
   def index
     @label = Category.label(filter_id)
     @photos = Photo.category_photos(filter_id).paginate(page: params[:page])
-    @categories = Category.all
+    @categories = Category.all.sample(7)
```
Comment (MowAlon) [inline app/controllers/photos_controller.rb:10]: Why are we only showing a sample of categories?

Comment (rasensio1) [inline app/controllers/photos_controller.rb:10]: Can add in a scrolling sidebar tomorrow maybe?
would be fun.

**Comment by MowAlon** [comment]
We need to have a place where you can select any category, and this is the only place right now, so maybe the short answer is to show all categories and just not fix the position of the menu.

**Comment by MowAlon** [comment]
So I'll merge, but I trust we'll solve this category thing for realz.

**Pull request merged**